### PR TITLE
chore: add app gif

### PR DESCRIPTION
**Pull request by marcossevilla**
## Description

- Add app gif.
- Small fix on location page AppBar's color.

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Chat Location
 
+![app][app_gif]
```
Comment (felangel) [inline README.md:3]: thoughts on moving this gif after the description?

Comment (marcossevilla) [reply README.md]: I took photobooth as an example but I don't mind 😅

Comment (felangel) [reply README.md]: Actually maybe just move it after the badges?

Comment (marcossevilla) [reply README.md]: agree

**Review by felangel — APPROVED** [review]

**Pull request merged**